src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,180 @@
+
+#### Comment on a data value
+
+You can add a comment to any data value. Comments can be useful to add more information about a value, like noting reasons why a value is unusually high or out of the normal range.
+
+To add a comment to a data value, [make sure the data details sidebar is open](), then click the *Add comment* button in the *Comment* section.
+
+![](resources/images/data_entry_beta/comment-add-button.png) 
+
+Once you've written a comment, click the *Save comment* button.
+
+If a data value already has a comment, you can edit it by clicking the *Edit comment* button below the comment.
+
+![](resources/images/data_entry_beta/comment-edit-button.png) 
+
+#### Minimum and maximum limits
+
+A data value can have a minimum and maximum limit. These limits won't allow someone to enter values outside of the minimum or maximum. 
+
+To add limits to a data value, [make sure the data details sidebar is open](), then click the *Add limits* button in the *Min and max limits* section. If possible, an average value will be shown to help you create reasonable limits.
+
+![](resources/images/data_entry_beta/limits-add-button.png) 
+
+Once you've added a minimum and maximum limit, click the *Save limits* button.
+
+If a data value already has limits set up, you can change them or remove them. To change the limits, click the *Edit limits* button. To remove the limits, click the *Delete limits* button.
+
+![](resources/images/data_entry_beta/limits-edit-button.png) 
+
+#### Historical data
+
+To learn more about a data value, you can see the last twelve values in the *History* section of the data details sidebar. [Make sure the data details sidebar is open](), then open the *History* section
+
+![](resources/images/data_entry_beta/value-history.png) 
+
+When working through a data entry form, the *History* section is closed by default to prevent too much network data from being sent for every value.
+
+#### Audit log
+
+Every data entry cell has an audit log that shows when values were changed and who changed them. To see the audit log, [make sure the data details sidebar is open](), then open the *Audit log* section.
+
+<!-- todo: add image -->
+
+When working through a data entry form, the *Audit log* section is closed by default to prevent too much network data from being sent for every value.
+
+
+### Printing
+
+To print a data entry form, click the *Options* button in the top bar. From the dropdown menu, you can choose to print the form including cell data values, or an empty form.
+
+![](resources/images/data_entry_beta/print.png) 
+
+## Working offline
+
+==WIP==
+
+
+## Features not supported in the beta version
+
+The **Data Entry (beta) app** is constantly evolving and new features are being added. There are some features that the previous Data Entry app offered that are supported yet:
+- **Custom forms** aren't supported yet, but we're learning more about how custom forms are used to build a sustainable, flexible solution.
```

Comment:
Custom forms are supported, but we don't support custom CSS and JS. So if the custom form is just a table created in the custom form editor in the maintenance app, it should work (see [here](https://debug.dhis2.org/dev/dhis-web-aggregate-data-entry/#/?attributeOptionComboSelection=LFsZ8v5v7rq-C6nZpLKjEJr_yY2bQYqNt0o-i4Nbp8S2G6A&dataSetId=lyLU2wR22tC&orgUnitId=DiszpKrYNg8&periodId=202209))!

Reply:
Ah, thanks for the clarification @Mohammer5. I've edited this to be clearer.